azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -446,6 +448,48 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							}, true),
 							DiffSuppressFunc: suppress.CaseDifference,
 						},
+						"load_balancer_profile": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							ForceNew: true,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"managed_outbound_ips": {
+										Type:          schema.TypeInt,
+										Optional:      true,
+										ValidateFunc:  validate.IntBetweenAndNot(1, 100, intsets.MaxInt),
```

Comment:
we should be able to make this:

```suggestion
										ValidateFunc:  validatation.IntBetween(1, 100),
```